test/unit/storage_async_test.rb

```diff
@@ -5,388 +5,390 @@
 class StorageAsyncTest < Test::Unit::TestCase
   include TestHelpers::Certificates
 
-  def test_basic_operations
-    storage = StorageAsync::Client.instance(true)
-    storage.del('foo')
-    assert_nil storage.get('foo')
-    storage.set('foo', 'bar')
-    assert_equal 'bar', storage.get('foo')
-  end
+  if configuration.redis.async
```

Comment:
I think alternatively, we could do:
```
  setup do
    omit_if(configuration.redis.async)
  end
```
and
```
  setup do
    omit_unless(configuration.redis.async)
  end
```
in the async tests file accordingly.

This will result in "omitted" log lines for each tests, and reporting N omissions in the results, but maybe that's not bad.

Actually, we can add a message so it's clearer:
```
  setup do
    omit_if(configuration.redis.async, 'skipping a sync test as async mode is enabled')
  end
```